libs/banzai-logging/config.jsonnet

```diff
@@ -0,0 +1,21 @@
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = [
+  { output: '3.17.3', version: '3.17.3' },
+];
+
+local crds = ['logging.banzaicloud.io_flows.yaml', 'logging.banzaicloud.io_clusterflows.yaml', 'logging.banzaicloud.io_clusteroutputs.yaml', 'logging.banzaicloud.io_outputs.yaml'];
```

Comment:
Can you add some linebreaks here? Makes future changes to this list easier to review.

Reply:
Sure, done!